### PR TITLE
customize admin site

### PR DESCRIPTION
**Pull request by hoangnh1211**
## Related Tickets
- https://edu-redmine.sun-asterisk.vn/issues/40144

## WHAT
- Customize admin site

## HOW
- N/A

## WHY
- N/A

## Evidence (Screenshot or Video)
- N/A

### CODE REVIEW

**Comment by hoangnh1211** [comment]
ready

File: shop/admin.py [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+    list_display = ('username', 'email', 'phone', 'address', 'create_at', 'update_at', 'is_active')
+
+
+@admin.register(Category)
+class CategoryAdmin(admin.ModelAdmin):
+    list_display = ('name', 'description', 'parent')
+    
+
+@admin.register(Product)
+class ProductAdmin(admin.ModelAdmin):
```
Comment (huongnt-2545) [inline shop/admin.py:15]: cho phép khi tạo product đồng thời tạo ảnh luôn nhé

File: shop/models.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ class CustomUser(AbstractUser):
 class Category(models.Model):
     name = models.CharField(max_length=254)
     description = models.CharField(max_length=254, null=True)
-    parent = models.ForeignKey('Category', on_delete=models.SET_NULL, null=True)
+    parent = models.ForeignKey('Category', on_delete=models.SET_NULL, null=True,blank=True)
```
Comment (huongnt-2545) [inline shop/models.py:48]:
```suggestion
    parent = models.ForeignKey('Category', on_delete=models.SET_NULL, null=True, blank=True)
```

**Review by huongnt-2545 — COMMENTED** [review]
Update ticket redmine: status=IN PROGRESS, spent time

**Comment by hoangnh1211** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**